src/did/web5-did.js

```diff
@@ -37,8 +38,18 @@ class Web5DID {
     return this.#didConnect.permissionsRequest;
   }
 
+  get manager() {
```

Comment:
this means that every time someone calls `web5.did.manager` they're gonna get a new object.  can we avoid this?  it seems very unnecessary

also, `DidManager` seems kinda pointless since all it really does is just call methods on the `store`.  do we really need that collection of functions instead of just `this.didStore.thing(arg)`?

Reply:
`// Set Managed DID as running in browser memory and for which keys are available to sign messages.
    web5.did.manager.set(myDid.id, {`

that looks pretty nice and explanatory to me - keeping things namespaced.

Reply:
to be clear, im not opposed to the idea of `web5.did.manager`.  im just commenting on how it is implemented.

for example, instead of
```
clear: (...args) => DidManager.clear(...args, this.didStore),
```
why not just
```
clear: (...args) => this.didStore?.clear()
```

IMO ideally we'd have a `class DIDManager` that's instantiated inside the `constructor` of `Web5DID` instead of creating a new object each time it's called

Reply:
> this means that every time someone calls web5.did.manager they're gonna get a new object. can we avoid this? it seems very unnecessary
This doesn't seem to be a major issue given how infrequently this function is called in the normal lifecycle of an application built with Web5 JS.

> also, DidManager seems kinda pointless since all it really does is just call methods on the store. do we really need that collection of functions instead of just this.didStore.thing(arg)?
There are more capabilities coming to DID Manager, including interaction with a Key Store in the future.

> IMO ideally we'd have a class DIDManager that's instantiated inside the constructor of Web5DID instead of creating a new object each time it's called
Little functional difference between these two implementations, but your suggestion is more consistent with how the rest of the Web5 JS lib is implemented.  Refactored in 3f4b8d4a885e2fc59d502621e1a0ac85056536f6